blt/blt.yml

```diff
@@ -231,6 +231,11 @@ validate:
       - files.php.tests
 
 tests:
+  cypress:
+    basePath: ${repo.root}/tests/cypress
```

Comment:
This should include a set of the tests we want to run as was done in https://github.com/NCIOCPL/cgov-digital-platform/blob/d7671e9d0ff82fe16aebe6b0c6eea2c3b6109904/blt/blt.yml#L237. 

What is being done in that branch is that it is listing a number of folders that contain features. Each of the folders run sequentially, but the features within that folder run in parallel. 

"Pre-test" scripts would come first.

Reply:
Done